commit c2670609d1777102d6e00c3d599f4547efdb7404 (refs/changes/62/5362/1)
Author: Angela Fong2 <angela.fong@joyent.com>
Date:   2019-01-14T14:36:27-08:00 (9 months ago)
    
    TRITON-1086 Docs: Fix incorrect coal image URLs and broken formatting in coal setup/readme instructions

diff --git a/README.md b/README.md
index 4cf6b4b..b39a94f 100644
--- a/README.md
+++ b/README.md
@@ -127,7 +127,7 @@ the [Joyent Manufacturing Database](https://docs.joyent.com/private-cloud/hardwa
 To install Triton, first download the latest release image:
 
 ```bash
-curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/usb-latest.tgz
+curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/coal-latest.tgz
 ```
 
 Once you have downloaded the latest release image, you will need to
diff --git a/docs/developer-guide/coal-setup.md b/docs/developer-guide/coal-setup.md
index 9ffd912..e38a052 100644
--- a/docs/developer-guide/coal-setup.md
+++ b/docs/developer-guide/coal-setup.md
@@ -39,9 +39,9 @@ The CoaL tarball is approximately 2 GB and will take some time to download.
 
 To download the latest CoaL build:
 
-	```
-	curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/release-20180927-20180927T130527Z/headnode/coal-release-20180927-20180927T130527Z-g34dbd8a-4gb.tgz
-	```
+```
+curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/coal-latest.tgz
+```
 
 While you wait for the tarball download to complete, you can configure and install VMware.
 
@@ -59,12 +59,12 @@ Most Joyent team members run CoaL from VMware Fusion. We recommend VMware versio
 5. To configure CoaL's external and admin networks:
 
 	```bash
-	curl -s https://raw.githubusercontent.com/joyent/sdc/master/tools/coal-mac-vmware-setup | sudo bash
+	curl -s https://raw.githubusercontent.com/joyent/triton/master/tools/coal-mac-vmware-setup | sudo bash
 	```
 
 	**Note**: CoaL network setup scripts are available for Windows and Linux.
 
-	For Windows, download https://raw.githubusercontent.com/joyent/sdc/master/tools/coal-windows-vmware-setup.bat and run `coal-windows-vmware-setup.bat`.
+	For Windows, download https://raw.githubusercontent.com/joyent/triton/master/tools/coal-windows-vmware-setup.bat and run `coal-windows-vmware-setup.bat`.
 
 	On Linux, run:
 	```bash
@@ -93,27 +93,31 @@ Most Joyent team members run CoaL from VMware Fusion. We recommend VMware versio
 
 **Note**: If you are working from the command line instead of the VMware GUI, run:
 
-	```
-	open coal-<branch>-<build_date_time>-<git_sha1_hash>-4gb.vmwarevm
-	```
+```
+open coal-<branch>-<build_date_time>-<git_sha1_hash>-4gb.vmwarevm
+```
 
 **Note**: To configure memory:
-    1. Select **Virtual Machine** and then click **Settings**.
-	The **USB-headnode Settings** dialog displays.
-    2. Click **Processes & Memory**. Change the memory settings, if needed.
-	**Note**: The default memory is set to 8192 MB. You can change this setting, but *be sure to leave Mac OS X with at least 8 GB to avoid resource allocation problems*.
-	If you are not using the VMware GUI, you can change the default memory in the CoaL `USB-headnode.vmx` file.  You can set memsize = "6144", or according to your preference.
-3. If you changed the memory settings, restart VMware.
+
+1. Select **Virtual Machine** and then click **Settings**.
+
+   The **USB-headnode Settings** dialog displays.
+2. Click **Processes & Memory**. Change the memory settings, if needed.
+
+   **Note**: The default memory is set to 8192 MB. You can change this setting, but *be sure to leave Mac OS X with at least 8 GB to avoid resource allocation problems*.
+
+   If you are not using the VMware GUI, you can change the default memory in the CoaL `USB-headnode.vmx` file.  You can set memsize = "6144", or according to your preference.
+3. If you changed the memory settings, restart the VM.
 
 The GRUB boot loader menu displays.
 
-##GRUB menu navigation alert
+## GRUB menu navigation alert
 
 GRUB intervenes in the boot process, enabling you to specify the environment to load. The default is for the console to use ttyb. This is fine for production but not for our use.
 
 Immediately after highlighting **Live 64-bit** using the arrow keys, press **c** quickly in order to direct the console to use **vga**.
 
-	![CoaL Grub Boot Menu](../img/coal-grub-menu.png)
+![CoaL Grub Boot Menu](../img/coal-grub-menu.png)
 
 If you are too slow, the **Live 64-bit** option loads instead of the GRUB command line. If needed, press **Esc** to redirect the console back to the GRUB menu and try again.
 
@@ -130,11 +134,11 @@ When you are prompted with the GRUB menu:
 
 **Note**: If you see a blank screen with a cursor at the top left while the head node is booting, you may have forgotten to redirect the console. If so, press Esc to return to the GRUB menu.
 
-     ![kvm warning on boot](../img/coal-only-cursor.png)
+![kvm warning on boot](../img/coal-only-cursor.png)
 
 **Note**: Because you are in a virtual environment without Intel VT-x support enabled, you'll receive cpu and kvm warnings. You can safely ignore them:
 
-     ![kvm warning on boot](../img/coal-boot-warnings.png)
+![kvm warning on boot](../img/coal-boot-warnings.png)
 
 ## Configuring the head node
 
